test/release/examples/README

```diff
@@ -63,11 +63,14 @@ subdirectories by simply using:
 Other files
 -----------
 
-./
-  Makefile        : Makefile for compiling the examples
+./                : This directory
```

Comment:
Nice improvement! I'd like to see the following minor refinements:

* Is Logs/ always created in examples/ ?  If not, the Logs/ line should be under "This directory and/or subdirectories".

* Instead of these two lines:

./                : This directory
./**              : This directory and/or subdirectories

I prefer these, respectively:

This directory:
This directory and subdirectories: